src/IniWriter.php

```diff
@@ -110,7 +110,7 @@ public function writeToString(array $config, $header = '')
     private function encodeValue($value)
     {
         if (is_bool($value)) {
-            return (int) $value;
+            return 'true';
```

Comment:
@ssm2017 sorry for the extremely late  reply on this. Do you maybe remember why this was changed? I think we would like to keep `(int) $value`

Reply:
@ssm2017 be good if you could let us know whether you're still working on this PR or not.

Reply:
hello
i do not use this code anymore.
i was using your library for a custom php code.
i do not remember why i have submitted this pr.

Reply:
Cheers for letting us know 👍